build.gradle

```
@@ -66,7 +66,7 @@ compileTestKotlin {
 dependencies {
     api platform('com.exactpro.th2:bom:3.0.0')
 
-    implementation 'com.exactpro.th2:common:4.0.0-th2-2150-books-pages-1520135874-SNAPSHOT'
+    implementation 'com.exactpro.th2:common:4.0.0-th2-2743-1530269241-SNAPSHOT'
```

```suggestion
    implementation 'com.exactpro.th2:common:4.0.0-th2-2743-*'
```
This approach can be used for brunch for intermediate development

---

`4.0.0-th2-2743-*` doesn't work for me. Looks like I have local `4.0.0` version after `publishToMavenLocal` task.
And with `implementation 'com.exactpro.th2:common:4.0.0'` I have next error:
`Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.5.1, expected version is 1.1.16.`
I'll try. Thanks!